roles/openshift_node/tasks/journald.yml

```diff
@@ -25,7 +25,9 @@
 # I need to restart journald immediatelly, otherwise it gets into way during
 # further steps in ansible
 - name: Restart journald
-  command: "systemctl restart systemd-journald"
```

Comment:
I think we're using command here due to retry logic.  I cannot remember why specifically.

Reply:
There seem to be other cases where we have retry logic around the service module so I don't think that's it. Though are we sure the output registers rc?

Reply:
I don't know if rc is registered or not.